### DISCUSSION TITLE
Need instruction and advice for building a new PKSim population.

### DISCUSSION THREAD

**Discussion by regulus92** [Open-Systems-Pharmacology/Forum#803] [Questions & Problems]
Hello. Here I intend to creat  a new population for PKSim, i.e., Chinese Han population. Since I am still learning Mobi, may I get some advices from you guys? Any comment welcome. Thank you.

**Comment by AndreDlm** [comment]
Hi @regulus92 , 

in MoBi you can only simulate one individual, but not populations (which is what you want to do, I assume?). If you are interested in population simulations, you need to do this in PK-Sim.

You may use the ICRP Extended population as a kind of "dummy" population in which change the anatomical and physiological parameters so that they reflect those observed in, e.g. a Chinese Han population. You can find more info [here](https://github.com/Open-Systems-Pharmacology/ICRP-Extended)

Maybe this helps.

**Comment by PavelBal** [comment]
If I understood correctly, @regulus92 wants to contribute by _developing_ new population, i.e., providing data on physiology etc?

**Comment by regulus92** [comment]
@PavelBal Exactly

**Comment by msevestre** [comment]
@regulus92 What's the status here ? Do you need some guidance?
